css/navbar.css

```
@@ -102,6 +104,49 @@ nav li a:hover {
   margin-right: 10px;
 }
 
+.dropdown {
+  width: 180px;
+  background-color: var(--color-white);
+  border: 2px solid var(--color-black);
+  border-radius: 0.5rem;
+  color: var(--color-black);
+  position: absolute;
+  top: 58px;
+  right: 16px;
+  z-index: 100;
+}
+
+.dropdown-list {
+  padding: 0;
+  margin: 0;
+}
+
+.dropdown-link {
+  color: var(--color-black);
+}
+
+.line {
+  border: none;
+  border-bottom: 2px solid var(--color-black);
+  margin: 0;
+}
+
+.dropdown-item {
+  padding: 1rem 1rem;
```

Can we please combine the properties 
```suggestion
  padding: 1rem;
```